docs/developer-guide.md

```diff
@@ -41,6 +41,10 @@ executable binaries:
 
 ![architecture image](img/VCF-validator-architecture.png)
 
+In case of dependent libraries being installed in non-standard paths,
+update LIBRARY_PATH as shown below
+  'export LIBRARY_PATH=$LIBRARY_PATH:<path to required library's libpath>'
+
```

Comment:
I'm not sure in which case this applies. 
I've not seen this being used anywhere in the `CMakeList.txt` and it didn't alleviate my compilation issues.

Reply:
This could be useful when dependent libraries, expected to be available through OS,  were installed in paths which are not part of library search. During Mac OS builds, I used to get link issues with zstd and icu libraries. They were present inside home-brew folders which were not part of any lib search environment variables.

Tried different ways to resolve them, like modifying CMake file, updating home-brew/cmake variables etc., and later noticed this from build.yml, as the easiest solution. Having this in dev-guide may help those building in Mac where many of the dependencies may be on a different path. (This seem to be applicable for linux builds as well but usually not required as installations usually made in standard search paths).